pandas/io/excel/_base.py

```diff
@@ -1014,16 +1014,22 @@ def close(self):
         return content
 
 
-XLS_SIGNATURE = b"\xD0\xCF\x11\xE0\xA1\xB1\x1A\xE1"
-ZIP_SIGNATURE = b"PK\x03\x04"
-PEEK_SIZE = max(len(XLS_SIGNATURE), len(ZIP_SIGNATURE))
+SIGNATURES = {
+    "biff2": b"\x09\x00\x04\x00\x07\x00\x10\x00",
+    "biff3": b"\x09\x02\x06\x00\x00\x00\x10\x00",
+    "biff4": b"\x09\x04\x06\x00\x00\x00\x10\x00",
+    "biff5": b"\xD0\xCF\x11\xE0\xA1\xB1\x1A\xE1",
+    "biff8": b"\xD0\xCF\x11\xE0\xA1\xB1\x1A\xE1",
```

Comment:
Does the biff version matter? I think no - in which case can you make this a tuple with comments, e.g.

```
BIFF_SIGNATURES = (
    ...
    b"\xD0\xCF\x11\xE0\xA1\xB1\x1A\xE1",  # biff5 and biff8
)
ZIP_SIGNATURE = ...
```